### PR TITLE
Add huge services to load test

### PR DESCRIPTION
**Pull request by tosi3k**
This is to add a way of testing the scalability of EndpointSlice objects.

/sig scalability

### CODE REVIEW

File: clusterloader2/testing/load/config.yaml [hunk 1]
```diff
@@ -428,6 +441,9 @@ steps:
         Group: scheduler-throughput
         CpuRequest: 1m
         MemoryRequest: 10M
+{{if $ENABLE_HUGE_SERVICES}}
+        SvcName: huge-service
```
Comment (wojtek-t) [inline clusterloader2/testing/load/config.yaml:445]: What is this used for?

Comment (tosi3k) [reply clusterloader2/testing/load/config.yaml:445]: Needed [here](https://github.com/kubernetes/perf-tests/pull/1681/files#diff-c83abe3d6dd61012c9df0cdc289fad5c771e1f8c634f622e4285b07563870068R19-R21) as label based on which the huge service [selects](https://github.com/kubernetes/perf-tests/blob/e34912ed01fda93e0e054a544e107f443498ab9e/clusterloader2/testing/load/service.yaml#L13) pods to front.

**Pull request merged**